src/app/thirdparty/libs/angular-inview.js

```diff
@@ -0,0 +1,387 @@
+// # Angular-Inview
```

Comment:
should not commit this file take from bower components
use **bower install angular-inview**

Reply:
I had used bower install angular-inview but the problem is with one arrow function which is there in angular-inview.js file which was throwing error then we make changes there and move this file in lib.